tests/dsp/test_build_dsp.py

```diff
@@ -0,0 +1,77 @@
+import os
+from pathlib import Path
+
+import pytest
+
+from pygama import lgdo
+from pygama.dsp import build_dsp
+from pygama.lgdo.lh5_store import LH5Store, ls
+from pygama.raw import build_raw
+
+config_dir = Path(__file__).parent/'configs'
+
+
+@pytest.fixture(scope="module")
+def multich_raw_file(lgnd_test_data):
+    out_file = '/tmp/L200-comm-20211130-phy-spms.lh5'
+    out_spec = {
+      'FCEventDecoder': {
+        'ch{key}': {
+          'key_list': [[0, 6]],
+          'out_stream': out_file + ":{name}",
+          'out_name': 'raw'
+        }
+      }
+    }
+
+    build_raw(in_stream=lgnd_test_data.get_path('fcio/L200-comm-20211130-phy-spms.fcio'),
+              out_spec=out_spec, overwrite=True)
+
+    return out_file
+
+
+def test_build_dsp_basics(lgnd_test_data):
+    build_dsp(lgnd_test_data.get_path('lh5/LDQTA_r117_20200110T105115Z_cal_geds_raw.lh5'),
```

Comment:
I think I'm missing something here...What kind of object is `lgnd_test_data` and how is it getting passed here? I assume this is pointing to the test data repository?

Reply:
It's defined [here](https://github.com/legend-exp/pygama/blob/main/tests/conftest.py#L5-L9) as a [Pytest fixture](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html). Fixtures are useful to share objects between tests.